src/condor_utils/job_ad_instance_recording.cpp

```diff
@@ -107,6 +107,36 @@ initJobEpochHistoryFiles(){
 	}
 }
 
+
+classad::ClassAd *
+copyEpochJobAttrs( const classad::ClassAd * job_ad, const classad::ClassAd * other_ad, const char * banner_name ) {
+    std::string paramName;
+    formatstr( paramName, "%s_JOB_ATTRS", banner_name );
+
+    // For admins to explicitly specify no attributes, these three
+    // parameters must NOT be in the param table.
+    if(! param_defined_by_config(paramName.c_str())) {
+        if( (strcmp(banner_name, "INPUT" ) == 0) ||
+          (strcmp(banner_name, "OUTPUT" ) == 0) ||
+          (strcmp(banner_name, "CHECKPOINT" ) == 0) ) {
```

Comment:
I'm cool if you keep this as is, but I personally have a preference for using the `MATCH` macro for `strcmp()`.